yang/subdir.am

```diff
@@ -48,4 +49,7 @@ endif
 
 if VRRPD
 dist_yangmodels_DATA += yang/frr-vrrpd.yang
+
+if ZEBRA
+dist_yangmodels_DATA += yang/frr-zebra.yang
```

Comment:
You also need to update the `frr_native_modules` global variable with this new module.